code/modules/economy/account.dm

```diff
@@ -118,7 +122,17 @@
 /datum/bank_account/department/New(dep_id, budget)
 	department_id = dep_id
 	account_balance = budget
-	account_holder = SSeconomy.department_accounts[dep_id]
+	var/list/total_department_list = SSeconomy.department_accounts+SSeconomy.nonstation_accounts
+
+	account_holder = total_department_list[dep_id]
+
 	SSeconomy.generated_accounts += src
 
+/datum/bank_account/proc/is_out_station_budget() // returns TRUE if the budget account isn't from the station. (like VIP one)
```

Comment:
Wait shouldn't this return true if it is out of the station budget rather than if it isn't?

Reply:
Yes, it is working as you said. If they're not the station budget, it returns false, because it failed to `if()` checks.

Reply:
Then the comment is incorrect

Reply:
` If they're not the station budget, it returns false,`

This was incorrectly spoken. I mean, it returns true if it's not station budget.